nservicebus/upgrades/support-policy.md

```diff
@@ -30,4 +30,13 @@ If a version is not listed or has expired but support is required [contact Parti
 
 ### NServiceBus 3.x
 
-As of 10 October 2015, Version 3 is no longer supported. If support is required [contact Particular](http://particular.net/contactus).
\ No newline at end of file
+As of 10 October 2015, Version 3 is no longer supported. If support is required [contact Particular](http://particular.net/contactus).
+
+
+## Compatibility guarantees
+
+The compatibility guarantees cover two aspects: upgrades and communication between endpoints in various versions (so called `wire compatibility`).
```

Comment:
Should we give these aspects both names? i.e. Upgrade compatibility and Wire compatibility.

Upgrade Compatibility says that you can always upgrade an endpoint to the next major. Skipping majors is never supported. Downgrades are not supported. It's one-way. One step at a time.

Wire Compatibility says that endpoints of 2 adjacent majors can communicate. It might be worth noting that majors can introduce new features that break Wire Compatibility but these are always opt in (we could probably do a better job of pointing these out).